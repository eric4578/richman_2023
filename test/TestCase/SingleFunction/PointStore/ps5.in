f 1000 10