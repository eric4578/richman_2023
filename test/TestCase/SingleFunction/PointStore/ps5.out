退出道具房
